google/cloud/datastore/helpers.py

```diff
@@ -490,6 +490,14 @@ def _set_protobuf_value(value_pb, val):
         setattr(value_pb, attr, val)
 
 
+def _set_database_id_to_request(request, database_id=None):
```

Comment:
Since this method is intended to be used outside this module and it actually is used in other modules, it should not be named with `_` at the start. In Python, things that are names with `_` prefix are considered "private", so should not be used outside of their local scope (in this case outside this file).

Please rename the method.

Reply:
Fixed it, thanks for spotting